conftest.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import cupy as cp
+import numpy as np
+import pytest
+from holoscan.core import Application, Fragment
+
+
+@pytest.fixture
+def app():
+    return Application()
+
+
+@pytest.fixture
+def fragment():
+    return Fragment()
+
+
+@pytest.fixture
+def config_file():
+    yaml_file_dir = os.path.dirname(__file__)
+    config_file = os.path.join(yaml_file_dir, "operator_parameters.yaml")
+    return config_file
+
+
+@pytest.fixture
+def mock_image():
+    def _factory(shape, dtype=cp.uint8, backend="cupy"):
+        if backend == "cupy":
+            return cp.random.randint(0, 255, size=shape, dtype=dtype)
+        elif backend == "numpy":
+            return np.random.randint(0, 255, size=shape, dtype=dtype)
+        else:
+            raise ValueError(f"Unknown backend: {backend}")
```

Comment:
Three suggested updates:
- use recommended new-style random [Generator](https://numpy.org/doc/2.2/reference/random/generator.html#random-generator) APIs.
- support floating point types via uniform distribution in [0.0, 1.0).
- allow specifying seed for the generator so tests can be made deterministic

```suggestion
    def _factory(shape, dtype=cp.uint8, backend="cupy", seed=None):
        if backend == "cupy":
            xp = cp
        elif backend == "numpy":
            xp = np
        else:
            raise ValueError(f"Unknown backend: {backend}")
        rng = xp.random.default_rng(seed)
        dtype = xp.dtype(dtype)
        if dtype.kind in 'ui':
            img = rng.integers(0, 256, size=shape, dtype=dtype, endpoint=False)
        elif dtype.kind in 'f':
            img = rng.uniform(0.0, 1.0, size=shape, dtype=dtype)
        else:
            raise ValueError(f"{dtype=} unsupported")
        return img
```